src/main/java/com/blackducksoftware/integration/hub/service/model/ScannerSplitStream.java

```diff
@@ -220,36 +221,42 @@ public void close() throws IOException {
     }
 
     private void writeToConsole(final String line) {
-        if (StringUtils.containsIgnoreCase(line, DEBUG) || StringUtils.containsIgnoreCase(line, TRACE)) {
+        String trimmedLine = line.trim();
+        if (doesStartWith(trimmedLine, DEBUG) || doesStartWith(trimmedLine, TRACE)) {
             // We dont want to print Debug or Trace logs to the logger
             return;
         }
         final StringBuilder outputBuilder = new StringBuilder();
         outputBuilder.append(output);
-        if (StringUtils.containsIgnoreCase(line, EXCEPTION)) {
-            // looking for 'Exception in thread' type messages
-            outputBuilder.append(line);
+        if (doesStartWith(trimmedLine, ERROR)) {
+            outputBuilder.append(trimmedLine);
             outputBuilder.append(System.getProperty("line.separator"));
-            logger.error(line);
-        } else if (StringUtils.containsIgnoreCase(line, FINISHED)) {
-            outputBuilder.append(line);
+            logger.error(trimmedLine);
+        } else if (doesStartWith(trimmedLine, WARN)) {
+            outputBuilder.append(trimmedLine);
             outputBuilder.append(System.getProperty("line.separator"));
-            logger.info(line);
-        } else if (StringUtils.containsIgnoreCase(line, ERROR)) {
-            outputBuilder.append(line);
+            logger.warn(trimmedLine);
+        } else if (doesStartWith(trimmedLine, INFO)) {
+            outputBuilder.append(trimmedLine);
             outputBuilder.append(System.getProperty("line.separator"));
-            logger.error(line);
-        } else if (StringUtils.containsIgnoreCase(line, WARN)) {
-            outputBuilder.append(line);
+            logger.info(trimmedLine);
+        } else if (StringUtils.containsIgnoreCase(trimmedLine, EXCEPTION)) {
+            // looking for 'Exception in thread' type messages
+            outputBuilder.append(trimmedLine);
             outputBuilder.append(System.getProperty("line.separator"));
-            logger.warn(line);
-        } else if (StringUtils.containsIgnoreCase(line, INFO)) {
-            outputBuilder.append(line);
+            logger.error(trimmedLine);
+        } else if (StringUtils.containsIgnoreCase(trimmedLine, FINISHED)) {
+            outputBuilder.append(trimmedLine);
             outputBuilder.append(System.getProperty("line.separator"));
-            logger.info(line);
+            logger.info(trimmedLine);
         }
 
         output = outputBuilder.toString();
     }
 
```

Comment:
I wonder if:
return line.toUpperCase().startsWith(string.toUpperCase())
might be better. You can even skip the second toUpperCase since you only call the method with all-caps strings.